File: src/test/java/com/hank/StudentTest.java
package com.hank;

import com.hank.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {

    @Test
    void highestTest(){
        Student student = new Student("Strong",80,60);
        Assertions.assertEquals(80,student.highest());
    }

    @Test
    void averageTest(){
        Student student = new Student("Strong",80,60);
        Assertions.assertEquals((80+60)/2,student.getAverage());
    }
}

File: src/test/kotlin/com/hank/StudentTest.kt
package com.hank.kotlin

import com.hank.kolin.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {

    @Test
    fun highestTest(){
        val student = Student("Strong",80,90)
        Assertions.assertEquals(90,student.highest())
    }

    @Test
    fun averageTest(){
        val student = Student("Strong",80,90)
        Assertions.assertEquals((80+90)/2,student.getAverage())
    }

    @Test
    fun gradingTest(){
        val student = Student("Strong",80,90)
        Assertions.assertEquals('B',student.grading())
    }
}